index.html

```diff
@@ -15,6 +16,9 @@
     </script>
 </head>
 <body>
+<p>You will be redirected to the documentation below in 5 seconds. Note
```

Comment:
The HTML Meta refresh waits for 5 second though. If you shut off java-script in your browser you'll see this behavior. I could put the `p` tag into a <noscript> tag though for clarity